config/main.py

```diff
@@ -6485,6 +6485,42 @@ def polling_int(ctx, interval):
     except ValueError as e:
         ctx.fail("Invalid ConfigDB. Error: {}".format(e))
 
+def is_port_egress_sflow_supported():
+    state_db = SonicV2Connector(host='127.0.0.1')
+    state_db.connect(state_db.STATE_DB, False)
+    entry_name="SWITCH_CAPABILITY|switch"
+    supported = state_db.get(state_db.STATE_DB, entry_name,"PORT_EGRESS_SAMPLE_CAPABLE")
+    click.echo("switch_capability-egress-sflow {}".format(supported))
```

Comment:
Why do we need to print this line every time the CLI command is called?

Reply:
Will remove this print.